compiler/util/llvmDebug.cpp

```diff
@@ -34,10 +34,15 @@
 #include "llvmUtil.h"
 #include "llvm/ADT/ArrayRef.h"
 #include "genret.h" 
-#include "llvm/Support/Dwarf.h"
 #include "llvm/Support/raw_os_ostream.h"
 #include <iostream>
 #include "llvm/ADT/DenseMap.h"
+
+#if HAVE_LLVM_VER >= 50
+#include "llvm/BinaryFormat/Dwarf.h"
+#else
+#include "llvm/Support/Dwarf.h"
+#endif
```

Comment:
Shouldn't this be covered by the include of `llvmDebug.h` above?

Reply:
Good point.  llvmDebug.cpp includes several files that are already included by llvmDebug.h.  I will push a change to delete the duplicates from llvmDebug.cpp so we don't have to port the files to new versions of LLVM twice.